README.md

```diff
@@ -44,12 +44,15 @@ Using the `psql` utility, run these two setup scripts from the command line, lik
 ```sh
 psql -f db/scripts/pres_setup.sql postgres
 psql -f db/scripts/pres_test_setup.sql postgres
+bundle exec rails db:seed
+RAILS_ENV=test bundle exec rails db:seed
 ```
 
 These scripts do the following for you:
 * create the test and dev PostgreSQL users.
 * create the test and dev databases.
 * setup the needed ownership and permissions between the DBs and the users.
+* seed data into development and test environments
```

Comment:
for development do we need to run `bundle exec rake m2c:seed_catalog`?

Reply:
AFAIK, no.  There's `m2c:seed_all_roots` and `m2c:seed_root`, but no `m2c:seed_catalog`.  Neither is important *for general development*.  If you want to run them for a given purpose, of course you could.  In any case, that isn't really part of the PR, which just moves `db:reset` from an every-rspec-execution scope to a "first time setup" scope.